<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>即时聊天平台</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* 主窗口样式 */
    QWidget {
        background-color: #f0f5ff;
        font-family: 'Microsoft YaHei';
    }

    /* 标题样式 */
    #headerLabel {
        color: #1a3a5a;
        font-size: 24px;
        font-weight: bold;
        qproperty-alignment: AlignCenter;
    }

    /* 聊天显示区域 */
    QTextEdit#chatTextEdit {
        background-color: white;
        border: 2px solid #c5d9f1;
        border-radius: 10px;
        padding: 15px;
        font-size: 16px;
        color: #333333;
        selection-background-color: #4a90e2;
        selection-color: white;
        min-height: 500px;
    }

    /* 消息输入框 */
    QLineEdit#msgLineEdit {
        border: 2px solid #c5d9f1;
        border-radius: 10px;
        padding: 15px;
        font-size: 16px;
        color: #222222;
        background-color: white;
    }

    QLineEdit#msgLineEdit:focus {
        border: 2px solid #4a90e2;
    }

    /* 按钮通用样式 */
    QPushButton {
        background-color: #4a90e2;
        color: white;
        border: none;
        border-radius: 10px;
        padding: 14px 28px;
        font-size: 18px;
        font-weight: bold;
        min-width: 120px;
    }

    QPushButton:hover {
        background-color: #3a7bc8;
    }

    QPushButton:pressed {
        background-color: #2c6bb6;
    }

    /* 返回按钮特殊样式 */
    #returnButton {
        background-color: #8a8a8a;
    }

    #returnButton:hover {
        background-color: #7a7a8a;
    }

    /* 聊天容器样式 */
    #chatContainer {
        background-color: white;
        border-radius: 15px;
        padding: 25px;
        margin: 25px;
        box-shadow: 0 6px 12px rgba(0,0,0,0.12);
    }
   </string>
  </property>
  <widget class="QWidget" name="chatContainer" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>750</width>
     <height>900</height>

    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="headerLabel">
      <property name="text">
       <string>即时聊天平台</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="recerverLabel">
      <property name="styleSheet">
       <string notr="true">
        font-size: 18px;
        color: #444444;
        qproperty-alignment: AlignCenter;
       </string>
      </property>
      <property name="text">
       <string>您正在和 xxx (xx) 沟通：</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTextEdit" name="chatTextEdit">
      <property name="lineWrapMode">
       <enum>QTextEdit::WidgetWidth</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="msgLineEdit">
      <property name="placeholderText">
       <string>请输入消息内容...</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="returnButton">
        <property name="styleSheet">
         <string notr="true">#returnButton</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
